### ISSUE TITLE
Segmentation fault in message mode

### ISSUE THREAD

**Issue by Dimonich** [nils-ohlmeier/sipsak#95]
When trying to use sipsak in message mode, Segmentation fault occurs.

How to reproduce:
Use command sipsak -s sip:1234@[ip address] -M -B Hello World!

Result:
![изображение](https://github.com/nils-ohlmeier/sipsak/assets/6535372/9eb98f42-056a-47fa-96da-357d6d0e28d0)

**Comment by nils-ohlmeier** [comment]
That is interesting. I can't reproduce locally here.

Could you please add a `-vvv` to the command line? That should tell us if it crashes before sending anything, or if sipsak actually receives a reply which makes it crash.

Is this on Linux? And did you compile sipsak yourself, or are you using a binary provided by your distro?

**Comment by Dimonich** [comment]
Yes, this is running on Linux VM(Rocky 9.3 Lite), I compiled sipsak myself.
Here is output with -vvv enabled:
![изображение](https://github.com/nils-ohlmeier/sipsak/assets/6535372/8d81c438-52a2-47f2-a844-d067865fcbe7)
